packages/sync-actions/src/utils/action-map-custom.js

```diff
@@ -26,21 +26,32 @@ const extractTypeFields = (diffedFields, nextFields) =>
     : nextFields
 const extractFieldValue = (newFields, fieldName) => newFields[fieldName]
 
-export default function actionsMapCustom(diff, newObj, oldObj) {
+export default function actionsMapCustom(
+  diff,
+  newObj,
+  oldObj,
+  customActions = {}
+) {
   const actions = []
+  const priceId = oldObj.id || undefined
+  if (customActions.setCustomType)
+    Actions.setCustomType = customActions.setCustomType
+  if (customActions.setCustomField)
+    Actions.setCustomField = customActions.setCustomField
```

Comment:
add customAction as param, so i can override the default actions

Reply:
Good idea 👍